docs/content.zh/docs/concepts/architecture.md

```diff
@@ -26,55 +26,55 @@ under the License.
 
 # Architecture
```

Comment:
It's not clear to me whether the title should be translated

Reply:
> It's not clear to me whether the title should be translated

hi, @caicancai 
Thx for the pr. 

It would be better to translate it like other translated pages~

Reply:
Thank you for your reply, done